### ISSUE TITLE
Can this work with non-public sheet?

### ISSUE THREAD

**Issue by tonydiep** [vivek-kasture/google-spreadsheet-demo#3]
Can this work with non-public Google sheets?
Is it possible to make sure the user is authenticated and is authorized to read / write to the spreadsheet?

**Comment by itsvick** [comment]
Yes, It will be work for private Google sheets. But In order to do that, we need some changes in it so any authorized user can access this sheet. @tonydiep
